lexactivator.go

```diff
@@ -38,11 +48,21 @@ func licenseCallbackWrapper(status int) {
 
 //export releaseUpdateCallbackWrapper
 func releaseUpdateCallbackWrapper(status int) {
-	if releaseUpdateCallbackFuncion != nil {
-		releaseUpdateCallbackFuncion(status)
+	if legacyReleaseCallbackFunction != nil {
+		legacyReleaseCallbackFunction(status)
 	}
 }
 
+//export newReleaseUpdateCallbackWrapper
+func newReleaseUpdateCallbackWrapper(status int, releaseJson *C.char) {
+   releaseJsonStr := ctoGoString(releaseJson)
+   if releaseCallbackFunction != nil {
+      var release Release
+      json.Unmarshal([]byte(releaseJsonStr), &release)
```

Comment:
I forgot to add the check for empty releaseJsonStr. I will add that and update the pull request